mmv1/templates/terraform/custom_expand/resourceref_with_validation.go.tmpl

```diff
@@ -15,5 +15,15 @@ func expand{{$.GetPrefix}}{{$.TitlelizeProperty}}(v interface{}, d tpgresource.T
 	if err != nil {
 	return nil, fmt.Errorf("Invalid value for {{underscore $.Name}}: %s", err)
 	}
+	{{- if $.ResourceMetadata.IsTgcCompiler }}
```

Comment:
is what allows us to delete the logic from mmv1/templates/tgc_next/tfplan2cai/expand_property_method_tgc.go.tmpl ?